definitions.h

```diff
@@ -223,6 +223,20 @@ class BufferImage : public Buffer2D<PIXEL>
 class Attributes
 {      
     public:
+
+        // Image
+        double u;
+        double v; 
+        void* ptrImg;
```

Comment:
This is great! But i know that the teacher wanted us to go deeper and figure out something for our selves